### PR TITLE
fix up error where peer doesn't exist or has no connection

### PR DESCRIPTION
**Pull request by jpaussa**
Also tweak timeouts to something a bit more sane. #20

### CODE REVIEW

File: peerme/euroix_json.py [hunk 1]
```diff
@@ -61,15 +61,13 @@ def fetch_json(self, ixp_json_file, timeout=10):
             ) for url in ixp_data_urls
         ]
         completed_tasks, _ = self.loop.run_until_complete(
-            asyncio.wait(http_tasks, timeout=10)
+            asyncio.wait(http_tasks, timeout=30)
```
Comment (cooperlees) [inline peerme/euroix_json.py]: Lets used the passed in value as I should of originally 🗡

**Review by cooperlees — CHANGES_REQUESTED** [review]
Lets fix the use of the timeout value

File: peerme/euroix_json.py (hunk 1 above)
Comment (jpaussa) [reply peerme/euroix_json.py]: There isn't any passed in value... having a cli option is possible but kinda overkill, might add it as a config option (would rather not bloat the cli options).

File: peerme/euroix_json.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def __init__(self, config, refresh_data=False, loop=None):
 
         return url, data
 
-    def fetch_json(self, ixp_json_file, timeout=10):
+    def fetch_json(self, ixp_json_file, timeout=30):
```
Comment (cooperlees) [inline peerme/euroix_json.py]: ^^ Use the passed in timeout in this function

Another diff can actually pass it in from config ...

File: peerme/euroix_json.py (hunk 1 above)
Comment (cooperlees) [reply peerme/euroix_json.py]: asyncio.wait(http_tasks, timeout=timeout) is what I ment ...

**Review by cooperlees — CHANGES_REQUESTED** [review]
Further clarify

**Review by cooperlees — COMMENTED** [review]
Fucking hot

**Review by cooperlees — APPROVED** [review]
You bueaty

**Pull request merged**